nim CI broken on amd64:`sudo apt-fast update -qq` failed

### Example
https://dev.azure.com/nim-lang/Nim/_build/results?buildId=14167&view=logs&jobId=c6054849-8341-5e23-b888-79fd7ec95d3a&j=c6054849-8341-5e23-b888-79fd7ec95d3a&t=bf861bfa-5326-5445-c69a-858de2cd39dc

### Current Output
```
Starting: Install dependencies (amd64 Linux)
==============================================================================
Task         : Bash
Description  : Run a Bash script on macOS, Linux, or Windows
Version      : 3.182.0
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/bash
==============================================================================
Generating script.
========================== Starting Command Output ===========================
/bin/bash --noprofile --norc /home/vsts/work/_temp/cdacc666-aad7-4774-bbc4-040448459710.sh

sudo apt-fast update -qq
E: Failed to fetch https://packages.microsoft.com/ubuntu/16.04/prod/dists/xenial/main/binary-amd64/Packages  Writing more data than expected (1113755 > 1111551)
E: Some index files failed to download. They have been ignored, or old ones used instead.
##[error]Bash exited with code '100'.
Finishing: Install dependencies (amd64 Linux)
```


### Additional Information
1.5.1 e922d73dd6b74a0ea9e1588bca945662f0797a9c
I added `set -e` in https://github.com/nim-lang/Nim/pull/17327 among other fixes; maybe this needs more care eg:
```
sudo apt-fast update -qq || echo "failed but acceptable"
```

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -88,7 +88,8 @@ jobs:
         Pin-Priority: 1001
         EOF
 
-        echo_run sudo apt-fast update -qq
+        # echo_run sudo apt-fast update -qq
+        echo_run sudo apt-fast update -qq || echo "failed, see bug #17343"
         # `:i386` (e.g. in `libffi-dev:i386`) is needed otherwise you may get:
         # `could not load: libffi.so` during dynamic loading.
         DEBIAN_FRONTEND='noninteractive' \

